Navaids/FIXES_Old.txt

```diff
@@ -1,3 +1,17 @@
+;withdrawn 2023/03
+ALVIN N051.39.44.830 W002.40.42.000
+AMRAL N052.06.53.430 W002.54.32.530
+BADIM N051.38.37.890 W002.30.40.430
```

Comment:
Don't ask me why but BADIM is remaining in ENR 4.4, so per #4580 will still be defined in FIXES_UK.txt, so this would cause a duplicate fix definition error.
```suggestion

```